generic/tests/cfg/timedrift.cfg

```diff
@@ -77,25 +64,37 @@
             test_duration = 600
             smp = 2
             stop_time = 60
-            drift_threshold = 10
         - with_signal_stop:
-            # starting from Win2012 Windows no longer relies on number of injected RTC interrupts for time keeping,
-            # and the time drift is seen because "stop" monitor command also stops PM timer,
-            # so this is unsupported scenario from Win2012.
-            Windows:
-                only Win2000, WinXP, Win2003, WinVista, Win7, Win2008
             no RHEL.3.9, RHEL.4
             type = timedrift_with_stop
             stop_interations = 1
             stop_with_signal = yes
             stop_time = 60
-            drift_threshold = 10
-            drift_threshold_single = 3
-            variants:
-                - clock_host:
-                    rtc_clock = host
-                - clock_vm:
-                    rtc_clock = vm
+    variants:
+        - default_clock:
+            no with_signal_stop, with_stop
+        - clock_host:
+            only with_signal_stop, with_stop
+            # starting from Win2012 Windows no longer relies on number of injected RTC interrupts for time keeping,
+            # and the time drift is seen because "stop" monitor command also stops PM timer,
+            # so this is unsupported scenario from Win2012.
+            Windows:
+                only Win2000, WinXP, Win2003, WinVista, Win7, Win2008
+        - clock_vm:
+            only with_signal_stop, with_stop
+            rtc_clock = vm
+    variants:
+        - default_base:
```

Comment:
For windows, `default_base` is duplicate with `base_localtime`, and for linux, it is duplicate with `base_utc`.
Need be improved again, thanks!

Reply:
@yanan-fu 
Thanks very much.

Update the cfg file and delete default_clock and default_base.